researchapp/sync_api.py

```diff
@@ -0,0 +1,62 @@
+''' Library for interfacing with the "synchronizer".
+'''
+import requests
+
+
+class SyncExtension(object):
+    ''' Flask extension for interacting with the Synchronizer API.
+    '''
+    def __init__(self, app=None):
+        self.session = None
+        self.host = None
+
+        if app:
+            self.init_app(app)
+
+    def init_app(self, app):
+        ''' Init the extension.
+        '''
+        self.session = requests.Session()
+        self.host = app.config['SYNCHRONIZER_HOST']
+
+    def list_providers(self):
+        ''' List available providers.
+        '''
+        resp = self.session.get(self.host + '/providers')
```

Comment:
We don't need to address in this PR, but if the answer is "they're not authenticated yet," can you create an issue to track this?